restic-cleanup.sh

```diff
@@ -2,6 +2,9 @@
 source "/root/OT-Settings/config.sh"
 N1=$'\n'
 
+echo "Unlocking restic stale locks"
```

Comment:
Locks should not be a regular thing... Whats your thoughts on having this?

Reply:
I had trouble with locks due to having to redo the install several times on different vps, it's usually not a problem and having restic unlock automated shouldn't cause any conflicts either, really up to you, I'd leave it in case someone really needs the unlock, I needed it a few times personally.